modin/pandas/test/test_groupby.py

```diff
@@ -1271,6 +1279,44 @@ def test_agg_func_None_rename(by_and_agg_dict, as_index):
     df_equals(modin_result, pandas_result)
 
 
+@pytest.mark.parametrize(
+    "as_index",
+    [
+        True,
+        pytest.param(
+            False,
+            marks=pytest.mark.xfail_backends(
+                ["BaseOnPython"], reason="See Pandas issue #39103"
+            ),
+        ),
+    ],
+)
+@pytest.mark.parametrize("by_length", [1, 3])
+@pytest.mark.parametrize(
+    "agg_fns",
+    [["sum", "min", "max"], ["mean", "quantile"]],
+    ids=["reduction", "aggregation"],
```

Comment:
Based on what do you divide fns into reduction and aggregation? Having a look at pandas doc all they are aggregation fns.

Reply:
this splitting is about how Modin processes described functions, we have two different implementations for GroupBy: 
1. GroupbyReduce (`sum`, `min`, `max`, `count`, `size`, ...)
2. Aggregation (`mean`, `quantile`, `median`, ...)

Reply:
Okay, I see.